content/_guides/catalyst.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Catalysing and De-escalation
+category: chanop
+---
+
+People often use Libera.Chat to get help with something on their computer.
+Technical support channels are one of the things we're here for!
+
+If you have ever had a technical problem, you know how overwhelming it can be.
+Sometimes people seeking help will get very frustrated and might have a temper
+or be rude. Patient and efficient help can be vital in avoiding conflict in
+these situations.
+
+This guide will outline some good practices for both operators and helpers for
+interacting with people. It will also discuss some common situations and ideal
+ways of responding.
+
+## Only be "opped up" when necessary
+
+Most channels have a select few operators who are responsible for managing the
+culture of the channel. These are the people with `+o` flags. Most clients
+display a `@` next to their nick. Libera.Chat does not have "halfops" flags
+like some other networks.
+
+We recommend that operators in channels refrain from being "opped up" unless
+it is necessary for immediate channel management activities like changing a
+locked topic or removing a problematic user. We discourage using the `+O`
+flag, which automatically "ops up" the flagged user on join. There are other
+ways for [users to find out](faq#how-to-find-out-who-runs-a-channel) who runs
+the channel.
+
+Being opped up (or even voiced `+v` for manual or `+V` for automatic) in your
+channel means you appear above normal users in your channel's user list, which
+creates a visible hierarchy. While this might sound desirable, it can
+interfere with your ability to give support. People will see you as an
+authority figure and be wary of you. It's like the difference between having a
+conversation with a boss or teacher, and having a conversation with a
+co-worker or a classmate. Users may view your responses as orders, threats, or
+trick questions when you do not intend them to be.
+
+Channel Ops should avoid demonstrating hierarchy and interact as regular
+participants in their channels as much as possible. Try not to be someone who
+only appears when things are heated. Having a rapport with your users will
+make it easier to de-escalate heated situations, making it easier to avoid
+having to forcefully remove people from your community.
+
+### Why is #libera different?
```

Comment:
```suggestion
### Why is `#libera` different?
```

Styleguide

Reply:
oh wasn't aware ticks were legal in headers or i'd have done that